plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/actions/layout/ArrangeBorderNodesAction.java

```diff
@@ -69,12 +69,16 @@ protected ArrangeBorderNodesAction(final IWorkbenchPart workbenchpart) {
      * 
      * @param id
      *            The id of the action
-     * @param text
-     *            The label of the action
+     * @param textSelection
+     *            The label of the action when elements are selected
+     * @param textDiagram
+     *            The label of the action when the diagram is selected
      */
-    protected void initAction(final String id, final String text) {
```

Comment:
You added a parameter to this method. This class is in an exported package. Can you add a note to the release note?

Reply:
Yes, a separate commit for the release note is forthcoming. PR coming soon.